packages/components/tree-v2/src/tree.vue

```diff
@@ -56,7 +57,7 @@ const emit = defineEmits(treeEmits)
 
 const slots = useSlots()
 
-const itemSize = 26
+const itemSize = computed(() => props.itemSize)
```

Comment:
**Suggestion:** I suggest that we should avoid naming internal variables as same as `props`

Reply:
Updated.